hcl2helper/doc.go

```diff
@@ -0,0 +1,4 @@
+/*
+Package hcl2helper provides helper functions for parsing or getting hcl2 types to and from a Packer plugin config.
+*/
+package hcl2helper
```

Comment:
I decided to leave the `pkr` suffix out of the name because being inside the packer sdk should be enough to separate logic from other hcl2 helpers from another tool. If one needs some discrepancy, they can always override the name when importing.

Reply:
I agree; this is a good name.